charts/opentelemetry-demo/values.yaml

```diff
@@ -4,6 +4,25 @@ observability:
   jaeger:
     enabled: true
 
+default:
```

Comment:
@Frapschen can you add a comment above `default` that explains that items in this object will apply to all components?